__tests__/app.test.js

```diff
@@ -221,4 +220,57 @@ describe("GET /api/articles/:article_id/comments", () => {
         });
     });
   });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+
+  describe("200: Success responses", () => {
+    
+    test("updates votes for given article_id and returns the updated article", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 2 })
+        .expect(200)
+        .then(({body: {article}}) => {
+          expect(article.length).toBeGreaterThan(0)
+
+          expect(article[0].article_id).toBe(1);
+
+          expect(article[0]).toEqual(expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          }))
```

Comment:
We should know what the article_id and the votes are! 

Also there should only be one article returned, so the length check is not necessary. In your model return rows[0] so you just get back the article object.